### PR TITLE
asset-hash-fixes

### PR DESCRIPTION
**Pull request by meetdhananifynd**

### CODE REVIEW

File: src/lib/Theme.ts [hunk 1]
```diff
@@ -62,7 +62,10 @@ import {
 import { simpleGit } from 'simple-git';
 import { THEME_TYPE } from '../helper/constants';
 
-shortid.characters('0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ_@');
+const nanoid = customAlphabet(
```
Comment (Shivraj0) [inline src/lib/Theme.ts:65]: Did you confirm random characters count here ? How have we decided this ?

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -403,7 +406,7 @@ export default class Theme {
             const imageCdnUrl = await Theme.getImageCdnBaseUrl();
             const assetCdnUrl = await Theme.getAssetCdnBaseUrl();
             Theme.createVueConfig();
-            const assetHash = shortid.generate();
+            const assetHash = nanoid();
```
Comment (Shivraj0) [inline src/lib/Theme.ts:409]: What is the default hash length size ? Could see we can pass custom length to this function.

**Review by Shivraj0 — CHANGES_REQUESTED** [review]
Reviewed

File: src/lib/Theme.ts (hunk 1 above)
Comment (meetdhananifynd) [reply src/lib/Theme.ts:65]: yes if we publish 1 theme every hour ~2 thousand years or 17M IDs needed, in order to have a 1% probability of at least one collision.
by brute-force method.

File: src/lib/Theme.ts (hunk 2 above)
Comment (meetdhananifynd) [reply src/lib/Theme.ts:409]: yes we can pass, i have passed 9 size as currently size 9 is being used in every themebundle

**Review by Shivraj0 — APPROVED** [review]

**Pull request merged**